live-examples/css-examples/flexbox/flex-shrink.html

```diff
@@ -0,0 +1,36 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="flex-shrink">
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">flex-shrink: 1;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">flex-shrink: 2;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">flex-shrink: 3;</code></pre>
```

Comment:
Maybe these could be? Thoughts @rachelandrew @wbamberg 

1. `flex-shrink: 0;`
2. `flex-shrink: 1;`
3. `flex-shrink: 2;`